packages/server/test/e2e/0_component_tests_spec.js

```diff
@@ -54,4 +57,21 @@ describe('e2e component tests', () => {
       return snapshot('all tests results summary', runSummary)
     })
   })
+
+  it('fails and exits with correct code', function () {
+    return e2e.exec(this, {
+      project,
+      config: {
+        video: false,
+        experimentalComponentTesting: true,
```

Comment:
I thought that our flag was simply `componentTesting` ?

Reply:
Maybe this flag can be removed...

Reply:
I am actually unsure now - maybe this is not needed. I will look into it